.travis.yml

```diff
@@ -1,9 +1,19 @@
 language: python
+
 python:
   - "3.4"
+
 addons:
   postgresql: "9.4"
+
+before_install:
+    pip install codecov
+
+after_success:
+    codecov
+
 before_script:
   - psql -c 'create database "cfdm-unit-test";' -U postgres
   - pip install -r requirements.txt
+
 script: nosetests
```

Comment:
I think we need to tell nose to output coverage metrics for this to work, e.g. `nosetests --with-coverage --cover-package webservices`. Unless codecov is just magic.